test/evergreen/run_format_configs.sh

```diff
@@ -21,20 +20,102 @@ fi
 
 success=0
 failure=0
+running=0
+parallel_jobs=8
+cleanup=0
+PID=""
+declare -a PID_LIST
+
+usage() {
+	echo "usage: $0 "
+	echo "    -j <num_of_jobs>  number of jobs to execute in parallel (defaults to 8)"
+
+	exit 1
+}
+
+# Wait for other format runs.
+wait_for_process()
+{
+	while true ; do
+		for process in ${PID_LIST[@]};do
+			ps $process > /dev/null
+			ps_exit_status=$?
+
+			if [ ${ps_exit_status} -eq "1" ] ; then
+				# The process is completed so remove the process id from the list of processes.
+				PID_LIST=(${PID_LIST[@]/$process})
+
+				#wait for the process to get the exit status.
+				wait $process
```

Comment:
If the process is finished, I think this results in an error. If you look at https://github.com/wiredtiger/wiredtiger/blob/develop/test/format/format.sh#L395C2-L395C2 we would see if it is alive, then we would call wait which blocks until it is finished.

Reply:
@mickgraham-mongodb I first [implemented](https://github.com/wiredtiger/wiredtiger/pull/9344/commits/a124110fbfa904b80ce50b5bb6b220795117eff9) to call wait on the first process of the process list array but it blocks until it is finished (in case we end up waiting on a long-running task), so I changed the logic to check if any of the processes are finished. 

Executing the `wait` command after the process is finished does not result in an error. I just confirmed by randomly returning the exit status(SUCCESS/FAILURE) from the test/format binary and looks like the `wait` command after the process is finished returns the right exit status.

Reply:
I thought it could still fail as we do see a number of "not a child of this process" messages in older releases. The exit status can be important (e.g. OOM matters to diagnose a change). But its not absolutely critical. I'm happy to resolve this.